docs/src/components/HeroSection/HeroSection.scss

```diff
@@ -33,6 +33,7 @@
   }
 
   &__heading {
+    z-index: 1;
```

Comment:
Instead of doing this hack, could we place the `heading` element inside the `bpkdocs-hero-inner` div?

Reply:
That would mean setting the background colour using `rgba` instead of opacity on the entire view.

Reply:
I will try and see as remember we tried this yesterday and it didn't really work

Reply:
Going to leave it as it is right now as it solves the solution - but will revisit it soon as I hacked it around a lot when I removed this and it caused strange results